lib/dpl/provider/packagecloud.rb

```diff
@@ -64,8 +64,8 @@ def is_source_package?(filename)
       def get_source_files_for(orig_filename)
         source_files = {}
         glob_args = ["**/*"]
-        package = ::Packagecloud::Package.new(open(orig_filename))
-        result = @client.package_contents(@repo, package)
+        package = ::Packagecloud::Package.new(file: orig_filename)
```

Comment:
Please refrain from using keyword arguments. Our deployment runs on 1.9.3 right now, and I think this will break it.